packages/rxjs-read/src/lib/read-of.spec.ts

```diff
@@ -0,0 +1,38 @@
+import { from } from 'rxjs';
+import { TestScheduler } from 'rxjs/testing';
+import { Read } from './read';
+import { readOf } from './read-of';
+
+const testScheduler = new TestScheduler((actual, expected) => {
+  expect(actual).toEqual(expected);
+});
+
+describe('readOf', () => {
+  describe('when creating a new Read of a value', () => {
+    let read: Read<string>;
+    const expectedValue = 'value';
+
+    beforeEach(() => {
+      read = readOf(expectedValue);
+    });
+
+    it('should be created', () => {
+      expect(read).toBeDefined();
+    });
+
+    describe('and when subscribing', () => {
+      it('should observe the whole state', () => {
+        testScheduler.run(({ expectObservable, cold }) => {
+          const expected = cold('(a|)', { a: expectedValue });
+          expectObservable(from(read)).toEqual(expected);
+        });
+      });
+    });
+
+    describe('and when synchronously accessing', () => {
+      it('should return the whole state', () => {
```

Comment:
`given-when-then`

Reply:
And fix it everywhere.

Reply:
no

Reply:
but feel free to do it your self, if you think its worth your time